cmd/sealer/cmd/cluster/delete.go

```diff
@@ -35,82 +35,86 @@ var (
 	deleteAll         bool
 )
 
-// deleteCmd represents the delete command
-var deleteCmd = &cobra.Command{
-	Use:   "delete",
-	Short: "delete an existing cluster",
-	Long: `delete command is used to delete part or all of existing cluster.
-User can delete cluster by explicitly specifying node IP, Clusterfile, or cluster name.`,
-	Args: cobra.NoArgs,
-	Example: `
+var longDeleteCmdDescription = `delete command is used to delete part or all of existing cluster.
+User can delete cluster by explicitly specifying node IP, Clusterfile, or cluster name.`
+
+var exampleForDeleteCmd = `
 delete default cluster: 
 	sealer delete --masters x.x.x.x --nodes x.x.x.x
 	sealer delete --masters x.x.x.x-x.x.x.y --nodes x.x.x.x-x.x.x.y
 delete all:
 	sealer delete --all [--force]
 	sealer delete -f /root/.sealer/mycluster/Clusterfile [--force]
 	sealer delete -c my-cluster [--force]
-`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		var cf clusterfile.Interface
-		if clusterFile != "" {
-			var err error
-			cf, err = clusterfile.NewClusterFile(clusterFile)
+`
+
+// NewDeleteCmd deleteCmd represents the delete command
+func NewDeleteCmd() *cobra.Command {
```

Comment:
Why to define a New function instead of defining an instance value? @kakaZhou719